webApp/src/main/java/com/infoshare/controller/VolunteerController.java

```diff
@@ -49,7 +51,20 @@ public String createVolunteerFormDetails(@ModelAttribute("volunteerForm") Volunt
 
     @GetMapping("/search")
     public String searchForVolunteer(Model model) {
```

Comment:
może: searchForAvailableVolunteers ? -> początkowo jak testowałam to zapomniałam że to jest kryterium i zastanwaiłam się dlaczego 'nie działa'  ;)